biogeophys/FatesHydroWTFMod.F90

```diff
@@ -163,6 +165,36 @@ module FatesHydroWTFMod
      procedure :: dftcdpsi_from_psi => dftcdpsi_from_psi_cch
      procedure :: set_wkf_param     => set_wkf_param_cch
   end type wkf_type_cch
+  ! =====================================================================================
+  ! Type1 Smooth approximation of Clapp-Hornberger and Campbell (CCH) water retention and conductivity functions
+  ! Bisht et al. Geosci. Model Dev., 11, 4085–4102, 2018
+  ! =====================================================================================
+
+  ! Water Retention Function
+  type, public, extends(wrf_type) :: wrf_type_smooth_cch
+     real(r8) :: th_sat   ! Saturation volumetric water content         [m3/m3]
+     real(r8) :: psi_sat  ! Bubbling pressure (potential at saturation) [Mpa]
+     real(r8) :: beta     ! Clapp-Hornberger "beta" parameter           [-]
+     real(r8) :: scch_pu, scch_ps, scch_b2, scch_b3
+   contains
+     procedure :: th_from_psi     => th_from_psi_smooth_cch
+     procedure :: psi_from_th     => psi_from_th_smooth_cch
+     procedure :: dpsidth_from_th => dpsidth_from_th_smooth_cch
+     procedure :: set_wrf_param   => set_wrf_param_smooth_cch
+     procedure :: get_thsat       => get_thsat_smooth_cch
+  end type wrf_type_smooth_cch
+
+  ! Water Conductivity Function
+  type, public, extends(wkf_type) :: wkf_type_smooth_cch
+     real(r8) :: th_sat   ! Saturation volumetric water content         [m3/m3]
+     real(r8) :: psi_sat  ! Bubbling pressure (potential at saturation) [Mpa]
+     real(r8) :: beta     ! Clapp-Hornberger "beta" parameter           [-]
+     real(r8) :: scch_pu, scch_ps, scch_b2, scch_b3
```

Comment:
Please provide description(s) of these parameters

Reply:
same as above.